src/wp-admin/css/site-icon.css

```diff
@@ -75,3 +75,111 @@
 	flex-wrap: wrap;
 	gap: 10px;
 }
+
+.site-icon-preview {
+	display: flex;
+	height: 60px;
+	padding: 8px 0px 0px 8px;
+	align-items: flex-start;
+	gap: 8px;
+	position: relative;
+	overflow: hidden;
+	box-sizing: border-box;
+}
+
+.site-icon-preview.hidden {
+	display: none;
+}
+
+.site-icon-preview.customizer {
+	width: 275px;
+}
+
+.site-icon-preview.settings {
+	width: 350px;
+	padding: 16px 0px 0px 16px;
+	gap: 16px;
+	height: 88px;
+	margin-bottom: 20px;
+}
+
+  .site-icon-preview.settings .home-icon {
+	width: 58px;
+	height: 58px;
+	border-radius: 12.95px;
+  }
```

Comment:
```suggestion
.site-icon-preview.settings .home-icon {
	width: 58px;
	height: 58px;
	border-radius: 12.95px;
}
```

Reply:
Fixed this one manually since I needed to change the selector to fix cropping